.circleci/config.yml

```diff
@@ -34,17 +34,25 @@ commands:
           - v1-deps-{{ checksum "build.sbt" }}
 
       - run:
-          name: Compile
-          command: sbt test:compile
+          name: Compile 2.11
```

Comment:
It looks like the recommended approach is to split the `compile` and `test` tasks for the different Scala versions in independent CircleCI steps.

Reply:
okie dokie... we can also add a step for future 2.13 Sounds good !!

Reply:
Yeah! 2.13 is still not supported (I tried compiling locally against 2.13, and there are some code changes required to do so!). I'll create a separate ticket for doing so once this PR is merged. In any case, all of the changes here bring us a big step closer to a 2.13 release! :D